components/calendar/skin.css

```diff
@@ -139,3 +139,71 @@ governing permissions and limitations under the License.
     }
   }
 }
+@media (forced-colors: active) {
+  .spectrum-Calendar-date {
+    color: CanvasText;
+    forced-color-adjust: none;
+    --spectrum-calendar-button-icon-color: ButtonText;
+    --spectrum-calendar-day-background-color-cap-selected-down: Highlight;
+    --spectrum-calendar-day-background-color-cap-selected-hover: Highlight;
+    --spectrum-calendar-day-background-color-cap-selected: Highlight;
+    --spectrum-calendar-day-background-color-disabled: ButtonFace;
+    --spectrum-calendar-day-background-color-down: ButtonFace;
+    --spectrum-calendar-day-background-color-hover: Transparent;
+    --spectrum-calendar-day-background-color-key-focus: ButtonFace;
+    --spectrum-calendar-day-background-color-selected-hover: Transparent;
+    --spectrum-calendar-day-background-color-selected: Highlight;  
+    --spectrum-calendar-day-border-color-key-focus: ButtonText;
+    --spectrum-calendar-day-text-color-cap-selected-hover: HighlightText;
+    --spectrum-calendar-day-text-color-cap-selected: HighlightText;
+    --spectrum-calendar-day-text-color-disabled: GrayText;
+    --spectrum-calendar-day-text-color-hover: ButtonText;
+    --spectrum-calendar-day-text-color-key-focus: ButtonText;
+    --spectrum-calendar-day-text-color-selected-hover: HighlightText;
+    --spectrum-calendar-day-text-color-selected: HighlightText;
+    --spectrum-calendar-day-title-text-color: CanvasText;
+    --spectrum-calendar-day-today-background-color-down: ButtonText;
+    --spectrum-calendar-day-today-background-color-hover: ButtonText;
+    --spectrum-calendar-day-today-background-color-selected-hover: Highlight;
+    --spectrum-calendar-day-today-background-color: ButtonText;
+    --spectrum-calendar-day-today-border-color-disabled: GrayText;
+    --spectrum-calendar-day-today-border-color: ButtonText;
+    --spectrum-calendar-day-today-text-color-disabled: GrayText;
+    --spectrum-calendar-day-today-text-color: ButtonText;
+    &:not(.is-today) {
+      /* &:before {
+        content: none;
+      }  */
+    }
+    &.is-today {
+      /* &:before {
+        forced-color-adjust: auto;
+      } */
+    }
```

Comment:
Are we safe to remove these lines, or do we need to keep them around for future reference?

Reply:
please remove - leaving was an oversight

Reply:
@jnurthen ok, I just wanted to double-check. Would you mind removing these lines & pushing up that change? We can merge after that. Thanks!